.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_name:
+        description: The image name in the format <username>/<image>.
+        required: true
+        default: splatform/mits
+
+env:
+  IMAGE_NAME: "${{ github.event.inputs.image_name }}"
+  IMAGE_FILTER_REFERENCE: "${{ github.event.inputs.image_name }}"
+  REPOSITORY: "${{ github.repository }}"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Create new tag
+      run: ./build/ci/create_next_tag.sh
+    - name: Build image
+      run: ./build/build.sh
+    - name: Image registry login
+      run: |-
+        docker login \
+          --username "${{ secrets.DOCKER_USERNAME }}" \
+          --password-stdin <<<"${{ secrets.DOCKER_PASSWORD }}"
+    - name: Setup .netrc
+      run: ./build/ci/setup_netrc.sh
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    - name: Publish release
+      run: ./build/ci/publish.sh
```

Comment:
Hmm. I feel like it would be nice to split this script into multiple action steps:
- figure out the relevant tag (using a script)
- [create draft release](https://github.com/actions/create-release)
- [upload assets](https://github.com/actions/upload-release-asset)
- push image (as we do now)
- publish release (either via a shell script as now, or via [JS](https://github.com/actions/github-script))

The main advantage here would be that each step is easier to reason about (since they do less), and we can get the versioning step to print out what the result was.

Reply:
https://github.com/f0rmiga/minibroker-integration-tests/releases/tag/v0.3.0 was produced by the new approach.
Check https://github.com/f0rmiga/minibroker-integration-tests/runs/1059385597.